### ISSUE TITLE
Could not disable shadow route for `index` action in sails v0.12.3

### ISSUE THREAD

**Issue by sujanadiga-practo** [balderdashy/sails#4023]
**Sails version**: 0.12.3
**Node version**: 4.2.3
**NPM version**: 2.14.7
**Operating system**: Ubuntu 14.04

<hr/>

### Could not disable blueprint routes.

**Situation:** 
I have a UserController having index action and corresponding User model and I have bind 
`/guests` to UserController's index action. Now I want to disable blueprint route provided by sails, so that `/guests` should be the only endpoint for User index action. 
So I have set 
```
module.exports.blueprints = {
  actions: false,
  rest: false,
  shortcuts: false
};
```
However this seems to have no effect, and I could still list all users using `/user` endpoint.

I have created a [test project](https://github.com/sujanadiga-practo/sails-test) with the configuration I mentioned.

**Steps to reproduce**
1. Setup app
```
git clone https://github.com/sujanadiga-practo/sails-test
cd sails-test
npm install
sails lift
```
2. Go to `localhost:1337/user`
_Expected_: 404(since this route is not bound to any action in routes and blueprints are disabled)

_Obtained_:
```
{
  "message": "Users fetched",
  "users": []
}
```

**Comment by sujanadiga-practo** [comment]
<img width="795" alt="screen shot 2017-03-03 at 4 22 30 pm" src="https://cloud.githubusercontent.com/assets/16635045/23548494/c8cc108a-002d-11e7-83c7-3b1446ac86c9.png">

Also http method seems to have no effect

**Comment by mikermcneil** [comment]
@sujanadiga-practo thanks for the report- verified in latest sails 0.12.x.  We'll follow up with a patch momentarily.

**Comment by mikermcneil** [comment]
@sgress454 thanks!

**Comment by mikermcneil** [comment]
verified fixed in the repro app: https://trello.com/c/5oubdDY2/78-sails-0-12-x-blueprints-bug-4023-rather-than-supplying-any-kind-of-workaround-we-should-publish-a-0-12-13-patch-with-the-fix-i-v

**Comment by mikermcneil** [comment]
https://github.com/sujanadiga-practo/sails-test/pull/1

published in 0.12.13

**Comment by sujanadiga-practo** [comment]
Its fixed in v0.12.13, thank you @mikermcneil
